tracing-flame/src/lib.rs

```diff
@@ -369,9 +369,9 @@ where
     }
 }
 
-impl<S, W> Layer<S> for FlameLayer<S, W>
+impl<S, W> Subscribe<S> for FlameSubscriber<S, W>
```

Comment:
we shoould probably rename the `S` type params on `Subscriber`s but we can do that later I guess

Reply:
done